A1/js/questions.js

```diff
@@ -1,7 +1,17 @@
+import { delay } from './delay.js'
+
+
 export let questions = [
     { question: 'What\'s the biggest animal in the world?\n', correctAnswer: 'a', a: 'blue whale', b: 'pig', c: 'elephant', d: 'lion' },
     { question: 'Which country is brie cheese originally from?\n', correctAnswer: 'd', a: 'Austria', b: 'Germany', c: 'Italy', d: 'France' },
     { question: 'What year was Heinz established?\n', correctAnswer: 'c', a: '1900', b: '1903', c: '1869', d: '1788' },
     { question: 'What is the capital of Iceland?\n', correctAnswer: 'a', a: 'Reykjavík', b: 'Hafnarfjörður', c: 'Berlin', d: 'Kópavogur' },
     { question: 'Which planet is closest to the sun?\n', correctAnswer: 'c', a: 'Pluto', b: 'Earth', c: 'Mercury', d: 'Jupiter' }
 ];
+
+export async function getQuestions() {
+    const result = await delay();
```

Comment:
I think there is no need to assign the delay to a variable here as it always returns `undefined` and is not used anyways -0,5